stage0/src/zero_page.rs

```diff
@@ -116,6 +116,12 @@ impl ZeroPage {
         setup_data.header.next = self.inner.hdr.setup_data;
         self.inner.hdr.setup_data = &setup_data.header as *const oak_linux_boot_params::SetupData;
     }
+
+    /// Sets the address and size of the initial RAM disk.
+    pub fn set_initial_ram_disk(&mut self, ram_disk: &RamDiskInfo) {
+        self.inner.hdr.ramdisk_image = ram_disk.address.as_u64() as u32;
```

Comment:
Add a note here that we expect the address to always fit into 32 bits, as that's what QEMU uses via the fw_cfg API.

Reply:
Done.